backend/service/k8s/deployment.go

```diff
@@ -101,12 +101,144 @@ func ProtoForDeploymentSpec(deploymentSpec appsv1.DeploymentSpec) *k8sapiv1.Depl
 			resourceRequests[string(res)] = quantity.String()
 		}
 
+		LivenessProbeObject := &k8sapiv1.Probe{}
+		ReadinessProbeObject := &k8sapiv1.Probe{}
```

Comment:
lets move this probe generation code into its own function to keep this `ProtoForDeploymentSpec` smaller and lets add some unit test coverage for that new function.

Reply:
Move code to a new method and add test function